example/flaskr/templates/cookbook/index.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
```

Comment:
@leddigeb When this merges in, this template should give you a good start for the Cookbook page!

Reply:
Thanks Elaine as I was needing this. Will be working on this all day to close it out.